spec/operations/beta/project_board_operations_spec.rb

```diff
@@ -55,11 +55,11 @@
     end
 
     context 'when the project does not exist' do
-      it 'returns a failed response with the error' do
+      it 'returns a failed response with the error', :aggregate_failures do
```

Comment:
Doesn't it happen when _user_ doesn't exist, not the project?

Reply:
so, I had the same question, here I just made the test fit the changes.
 